eureka-webapp/src/main/java/edu/emory/cci/aiw/cvrg/eureka/servlet/proposition/EditPropositionServlet.java

```diff
@@ -62,9 +63,37 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         DataElement.Type propType = this.getPropTypeFromParam(req.getParameter("type"));
 		try {
 			List<FrequencyType> freqTypes = this.servicesClient.getFrequencyTypesAsc();
-			FrequencyType defaultFreqType = this.servicesClient.getDefaultFrequencyType();
+			/**FrequencyType defaultFreqType = this.servicesClient.getDefaultFrequencyType();**/
+                        
+                        //Find the default frequency
+                        FrequencyType defaultFreqType = null;
+                        for (FrequencyType freqType : freqTypes){
+                                if(freqType.isDefault()){
+                                        defaultFreqType = freqType;
+                                        break;
+                                }
+                        }                       
+                        if (defaultFreqType == null){
+                                throw new AssertionError(
```

Comment:
Good idea to check for no default! However, let's just let it pass through, since it's conceivable a database admin could unset the default, in which case it's no longer really programmer error. The client side needs to deal with there being no default set.